components/Sections/NewHeader/index.module.scss

```diff
@@ -0,0 +1,158 @@
+%navItemLi {
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  margin-left: var(--space-16);
+  white-space: nowrap;
+
+  @media (max-width: 900px) {
+    display: flex;
+    height: 42px;
+    margin-left: 0;
+
+    &:not(:last-child) {
+      margin-right: 0.5rem;
+    }
+  }
+
+  > a {
+    line-height: 0;
+  }
+
+  &:last-child {
+    margin-right: var(--space-08);
+  }
+}
+
+.header {
+  background: var(--color-fill-app);
+  border-bottom: var(--space-01) solid var(--black2);
+  position: sticky;
+  top: 0;
+  width: 100%;
+  z-index: 999;
+
+  .container {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0 var(--space-08) 0 var(--space-24);
+
+    @media (max-width: 900px) {
+      align-items: start;
+      flex-flow: wrap;
+      min-height: calc(1 * var(--nav-height));
+      padding: 0 var(--space-12);
+    }
+    .startWrapper {
+      align-items: center;
+      display: flex;
+      justify-content: flex-start;
+
+      .logo {
+        @media (max-width: 900px) {
+          align-items: center;
+          display: flex;
+          height: var(--nav-height);
+        }
+
+        svg {
+          height: var(--space-32);
+
+          @media (max-width: 900px) {
+            height: 2.4rem;
```

Comment:
Can we create variables for these mobile sizes?

Reply:
Do you want it in the PR or can it be as well on another one?

Reply:
This PR, as it is relevant to the implementation of the component itself.

Reply:
would you like the whole `@media (max-width: 900px)` to be a SCSS @mixin or a normal CSS var sitting in the tokens.scss file?

Reply:
Sounds like a good idea to become a mixin that can be wrapped : )

And if there's any other existing component with mobile specific styles we can update them in a follow-up PR to adopt those mixins!

Reply:
Put an section in tokens.scss named: "break point"

Reply:
Do we have a guide for it to follow or can I take [this one](https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/)?

Reply:
Feel free to use that one :)

Reply:
@ovflowd please take a look on my last push. I needed to write the mixins in the NewHeader scss file. Tried in a seperate file next to the tokens.scss and base.scss (I imported the file of course) and also in the index.scss itself but couldn't get it to run in storybook. Any idea what I was doing wrong?

Reply:
@u-rogel where did you import it?

Because the styles are imported here https://github.com/nodejs/nodejs.org/blob/major/website-redesign/.storybook/preview.tsx

But usually when you use mixins, you need to import the mixins on the file you're using them. If you create a new file called `mixins.scss` and import it within `styles/index.scss` it should probably work...

Wasn't it working?

Reply:
I think we should probably rename the files from here to become partials https://github.com/nodejs/nodejs.org/tree/major/website-redesign/styles https://sass-lang.com/guide#topic-4

Reply:
Take a look on the last commit. Is that what you mean? I would expect that importing it in the `styles/index.scss` would be enough but only importing it in the `index.module.scss` where I am using the mixins works for me.

Reply:
If you are ok with the current status, please remove the draft flag.